agent/config/config.go

```diff
@@ -314,7 +314,7 @@ func environmentConfig() Config {
 	imageCleanupDisabled := utils.ParseBool(os.Getenv("ECS_DISABLE_IMAGE_CLEANUP"), false)
 	minimumImageDeletionAge := parseEnvVariableDuration("ECS_IMAGE_MINIMUM_CLEANUP_AGE")
 	imageCleanupInterval := parseEnvVariableDuration("ECS_IMAGE_CLEANUP_INTERVAL")
-	numImagesToDeletePerCycle, err := strconv.Atoi(os.Getenv("ECS_NUM_IMAGES_DELETE_PER_CYCLE"))
```

Comment:
Can you just fix the warning instead?  It's actually important for this to have a default of `""` as the subsequent merges with `DefaultConfig()` and `fileConfig()` need to work.  If you make this not `""`, you break the assumptions of `Merge()`.